src/lib.rs

```diff
@@ -254,6 +288,7 @@ impl<'object, Endian> DebugInfo<'object, Endian>
         Ok(DebugInfo {
             debug_line: debug_line,
             units: units,
+            with_demangling: opts.with_demangling,
```

Comment:
Same as above, this should be a full `opts` (unless we decide to hoist the demangling code).